corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap5/widgets.js

```diff
@@ -108,8 +108,9 @@ hqDefine("hqwebapp/js/bootstrap5/widgets",[
         });
 
         _.each($(".date-picker"), function (input) {
+            throw new Error("Tempus Dominus is not yet working. See hqwebapp/js/bootstrap5/widgets to troubleshoot.");
```

Comment:
Is the idea that anyone converting a page to B5 would notice this JS error and realize they were not finished if a date picker is present?

Reply:
Yes. With the dependency commented out, the next line would fail anyway, I just figured this message is a little friendlier.